tools/extract_baserom.py

```diff
@@ -56,12 +56,17 @@ def main():
 
     args.output_dir.mkdir(parents=True, exist_ok=True)
     for dma_name, dma_entry in zip(dma_names, dma_entries):
-        if dma_entry.is_compressed():
+        if dma_entry.is_syms():
+            if (dma_entry.vrom_end - dma_entry.vrom_start) == 0:
+                continue
+            segment_rom_start = dma_entry.vrom_start
+        elif not dma_entry.is_compressed():
+            segment_rom_start = dma_entry.rom_start
+        else: # Segment compressed
```

Comment:
nit: flipping this conditional around so that `if dma_entry.is_compressed()` comes first would remove the need for this comment

Reply:
Might just be my own taste, but I like the error message being in the `else` branch. My eyes notice it better.